frontend/src/components/application/Releases.tsx

```diff
@@ -32,11 +32,20 @@ const Releases: FunctionComponent<Props> = ({ latestRelease }) => {
     [t],
   )
 
-  const releaseDescription = useMemo(
+  var releaseDescription = useMemo(
     () => (latestRelease.description ? latestRelease.description : noChangelog),
     [latestRelease.description, noChangelog],
   )
 
+  if (latestRelease.url) {
+    releaseDescription +=
+      "<a href='" +
+      latestRelease.url +
+      "' class='list-disc my-4 pl-10'>" +
```

Comment:
should probably add `target="_blank" rel="noreferrer"` ?

Reply:
Yes, but we probably shouldn't assume current browsers. At least the linting rules we have, don't do that.

https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-no-target-blank.md